### PR TITLE
`BEEFY` repo cleanup

### PR DESCRIPTION
**Pull request by adoerr**
- [x] Use Substrate integrated version of `BEEFY`
- [x] Delete moved code
- [x] Adjust README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,32 +50,16 @@ In addition to that the current status as well as a preliminary roadmap is prese
...
-`beefy-gadget` which is a POC of the BEEFY round logic. `beefy-pallet` which is mainly a thin
-integration layer over the session pallet and keeps track of the current authorities.
-Finally the BEEFY `primitives` crate which contains most of the type definitions for the
-BEEFY protocol.
+This project is an auxiliary repository for `BEEFY` utility, testing and example code. The `BEEFY` core components
+have been added to [Substrate](https://github.com/paritytech/substrate).
 
-The `primitives` crate also contains a test [light_client](.primitives/tests/light_client/) which demonstrates how BEEFY would
-be utilized by a light client implementation.
+This repository also used to keep track of `BEEFY` related issues, i.e. even issues regarding `BEEFY` core functionality, which is now part of [Substrate](https://github.com/paritytech/substrate), should be filed against **this** repository and **not** [Substrate](https://github.com/paritytech/substrate) itself.
```
Comment (tomusdrw) [inline README.md:56]:
```suggestion
This repository is also used to keep track of `BEEFY` related issues, i.e. even issues regarding `BEEFY` core functionality, which is now part of [Substrate](https://github.com/paritytech/substrate), should be filed against **this** repository and **not** [Substrate](https://github.com/paritytech/substrate) itself.
```

File: beefy-primitives/tests/basic_lc_operations.rs [hunk 2]
```diff
@@ -1,364 +0,0 @@
-// Copyright (C) 2020 Parity Technologies (UK) Ltd.
```
Comment (tomusdrw) [inline beefy-primitives/tests/basic_lc_operations.rs:1]: I think these tests are worth keeping around. AFAIR they were not part of the Substrate PR, so they should stay here.

**Review by tomusdrw — APPROVED** [review]

File: beefy-primitives/tests/basic_lc_operations.rs (hunk 2 above)
Comment (adoerr) [reply beefy-primitives/tests/basic_lc_operations.rs:1]: Yep, I'm thinking about adding a `light-client` folder and putting the tests there.

Comment (tomusdrw) [reply beefy-primitives/tests/basic_lc_operations.rs:1]: Yup, that makes a lot of sense. My initial idea was that the `light-client` test stuff can actually evolve into a proper light client implementation that we will be able to use on-chain eventually for bridges.

**Pull request merged**